Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -48,7 +49,7 @@
 validationLiteTracking.replace(globalValidation,globalValidationLiteTracking)
 validationLiteTracking.remove(condDataValidation)
 
-validationMiniAOD = cms.Sequence()
+validationMiniAOD = cms.Sequence(JetValidationMiniAOD * METValidationMiniAOD * type0PFMEtCorrectionPFCandToVertexAssociationForValidationMiniAOD * egammaValidationMiniAOD)
```

Comment:
if your modules don't use the trigger results they should go to the prevalidationMiniAOD